### PR TITLE
Added --package option to output subcommand

### LINKED ISSUE #820: Running code in Listing 8-2 results in error "Negative impls are not enabled in the current crate"
Repro steps:
- open https://book.cairo-lang.org/ch08-02-traits-in-cairo.html#negative-impls
- find Listing 8-2
- click "Run this code"
- error appears

### PR DESCRIPTION
**Pull request by supreme2580**
Closes #820

I added the --package option to output subcommand

So you can still run

```
cairo listings output
```
and 
```
cairo listings output --package <PACKAGE_NAME>
```

- - -
This change is [Reviewable](https://reviewable.io/reviews/cairo-book/cairo-book/922)

### CODE REVIEW

File: cairo-listings/src/main.rs [hunk 1]
```diff
@@ -38,10 +38,11 @@ fn main() {
     let cfg = &*CFG;
     let cfg_clone = cfg;
     let empty_arg = VerifyArgs::default();
+    let empty_package = OutputArgs::default();
```
[the remark this answers is not in the document]
Comment (supreme2580) [reply cairo-listings/src/main.rs:41]: Will remove it

File: cairo-listings/src/output.rs [hunk 2]
```diff
@@ -6,13 +6,31 @@ use std::{
 
 use regex::Regex;
 
-use crate::{cmd::ScarbCmd, config::VerifyArgs, run_command, utils::find_scarb_manifests, Config};
-
-pub fn process_outputs(cfg: &Config, arg: &VerifyArgs) {
-    let scarb_packages = find_scarb_manifests(cfg, arg);
+use crate::{cmd::ScarbCmd, config::VerifyArgs, config::OutputArgs, run_command, utils::find_scarb_manifests, Config};
+
+pub fn process_outputs(cfg: &Config, arg: &OutputArgs) {
```
[the remark this answers is not in the document]
Comment (supreme2580) [reply cairo-listings/src/output.rs:11]: Makes alot of sense, will do it

**Comment by supreme2580** [comment]
Hi @enitrat I have implemented the requested changes

**Review by enitrat — APPROVED** [review]
Great

**Pull request merged**